### PR TITLE
Order history

### PR DESCRIPTION
**Pull request by bbriannab**

### CODE REVIEW

File: Backend/index.js
Comment (Daniel-Lam1702) [inline Backend/index.js:1]: Consider returning more information of the orders such as menu items, price, and item components.

File: Front-End/src/pages/MenuBoard.css
Comment (Daniel-Lam1702) [inline Front-End/src/pages/MenuBoard.css:1]: Is there any need to make changes to the menu board for the order history?

**Review by Daniel-Lam1702 — CHANGES_REQUESTED** [review]

File: Front-End/src/pages/MenuBoard.css
Comment (bbriannab) [reply Front-End/src/pages/MenuBoard.css:1]: I wanted to reformat the menu board to left-align the text for each menu item.

Comment (Daniel-Lam1702) [reply Front-End/src/pages/MenuBoard.css:1]: Got it, for more organization even if it is a small change, it is better to make a separate branch for this. For next time! :)

**Review by Xach-A — APPROVED** [review]
looks good

**Review by Daniel-Lam1702 — APPROVED** [review]

**Pull request merged**